tests/39-basic-api_level.c

```diff
@@ -28,6 +28,7 @@ int main(int argc, char *argv[])
 {
 	int rc;
 	unsigned int api;
+	const unsigned int max_api = 2;
```

Comment:
Let's bump the `max_api` value up to something absurdly high, 1024 for example, and only test to make sure it is flagged as invalid.  Otherwise I fear this test is going to be a bit too fragile.